_episodes/format.md

```diff
@@ -3,51 +3,260 @@ title: "File format"
+>
+> 1. What is the format of the data?
+> 2. Do you know why the data is in that format?
+>
+{: .discussion}
+
+Here is a list of some common formats:
+
+- [NetCDF](https://www.unidata.ucar.edu/software/netcdf/docs/netcdf_data_model.html):
+  It was originally developed for the Earth science community, but it can be used for **many kinds of data**.
+  NetCDF views the world of scientific data in the same way that a geo-scientist might:
+  ![NetCDF]({{ page.root }}/fig/netcdf-classic-uml.png)
+
+  - There are various quantities such as temperature or elevation
+  located at points at particular coordinates in space and time.
+
+  - The quantities (here temperature or elevation) are stored as netCDF variables.
+  Variables are N-dimensional arrays of **data**, with a name.
+
+  - The coordinates information are stored as netCDF dimensions.
+  Dimensions have a length and a name.
+
+  - A scientist also wants to store supporting information or **metadata**, such as the units, or
+  some information about how the data were produced.
+  The metadata is stored as netCDF attributes that are always single values or one-dimensional arrays.
+
+> ## Why NetCDF format
+>
+> You work with some geospatial data for a project in your community.
+> You choose the NetCDF format for the data.
+> Your colleague asks:
+>
+> 1. Why did you choose the NetCDF format?
+> 2. Does this format help to make the data FAIR?
+>
+>> ## Solution
+>>
+>> 1. You found that NetCDF format is the standard in your research community and
+>> therefore interoperable and reusable for anyone.
+>> Also, the NetCDF format can contain both data and metadata.
+>>
+>> 2. Therefore, yes, the NetCDF format helps to make the data FAIR.
+> {: .solution}
+{: .challenge}
+
+- [GRIB](https://www.wmo.int/pages/prog/www/DPS/FM92-GRIB2-11-2003.pdf):
+  GRIB stands for general regularly-distributed information in binary.
+  It is **commonly used** by the [World Meteorological Organization](https://public.wmo.int/en)(WMO) for weather model data.
+
+  - GRIB coded data consist of a continuous bit-stream made of a sequence of
+  [octets](https://en.wikipedia.org/wiki/Octet_(computing)).
+  Thus, the representation of the data is **independent of** any particular machine representation.
+
+  - GRIB messages contain the **data** and **metadata**.
+
+> ## Why GRIB format
+>
+> You work with some geospatial data for a project in your community.
+> You choose the GRIB format for the data.
+> Your colleague asks:
+>
+> 1. Why did you choose the GRIB format?
+> 2. Does this format help to make the data FAIR?
+>
+>> ## Solution
+>>
+>> 1. You found that GRIB format is the standard in your research community and
+>> therefore interoperable and reusable for anyone.
+>> GRIB format is independent of any particular machine representation.
+>> Also, the GRIB format can contain both data and metadata.
+>>
+>> 2. Therefore, yes, the GRIB format helps to make the data FAIR.
```

Comment:
Although GRIB has long been the de facto WMO standard, I think the community is now moving away from it in many places. I *think* that is because netCDF is a more user-friendly (and maybe almost just as performant) alternative. So I would be careful to 'recommend' GRIB in this tutorial.

Reply:
I added an example of using the grib format to the text. We should be careful that "user-friendly" is not a measure that makes the data fair. At least I didn't see that FAIR resources refer to this. Also, I noticed that some resources provide two lists of formats: acceptable and recommended. Based on what we discussed, I am careful to not reject or ignore other commonly used formats in different communities.

Reply:
Well, actually I would argue that user-friendliness is an integral part of FAIR. I like to interpret the 'strive for FAIR data' as an endeavor to lower the barriers or thresholds for re-use by others. From that point of view, the easier it is for others to work with your data, the 'FAIRer' it is.